test/redis/store_test.rb

```diff
@@ -27,6 +27,28 @@
         @store.set(key, value, options)
       end
     end
+
+    describe 'with ex and nx' do
+      let(:key) { 'key' }
+      let(:value) { 'value' }
+      let(:options) { { nx: true, ex: 3600 } }
+
+      it 'must pass on options' do
+        Marshal.expects(:dump).times(4)
+
+        # without options no ex or nx will be set
+        @store.del(key)
+        @store.set(key, value, {}).must_equal 'OK'
```

Comment:
I found it odd, that the set with empty options is returning 'OK', but with certain options it returns true or false. Is this caused by this gem or the redis gem?